int64 tickCountA
int64 tickCountB
int64 distance

